components/ui-api-layer/Jenkinsfile

```diff
@@ -49,7 +44,7 @@ podTemplate(label: label) {
                             execute("./before-commit.sh ci")
                         }
 
-                        if (isMaster) {
+                        if (params.GIT_BRANCH == 'master') {
```

Comment:
According to other `Jenkinsfiles` it should be `params.FULL_BUILD`.

Reply:
Yes, that was a mistake, thanks for catching it!
Fixed.